Posts/2021/03/File-System-Watcher-Engine-Event.md

```diff
@@ -0,0 +1,110 @@
+# A Reusable File System Event Watcher for PowerShell
```

Comment:
You need to add the YAML front matter to this post. Please see https://github.com/PowerShell/Community-Blog/wiki/Creating-a-new-post